File: NDRefreshObjcExample/NDRefreshObjcExample/MultiView.m
//
//  MultiView.m
//  NDRefreshObjcExample
//
//  Created by Daisuke Fujiwara on 7/16/15.
//  Copyright (c) 2015 Nextdoor, Inc. All rights reserved.
//

#import "MultiView.h"

@implementation MultiView

+ (instancetype)newFromNib {
    NSArray *views = [[NSBundle bundleForClass:[self class]] loadNibNamed:@"MultiViewObjc" owner:self options:nil];
    return views.firstObject;
}

@end

File: NDRefreshSwiftExample/NDRefreshSwiftExample/MultiView.swift
//
//  MultiView.swift
//  NDRefreshSwiftExample
//
//  Created by Wenbin Fang on 7/16/15.
//  Copyright (c) 2015 Nextdoor, Inc. All rights reserved.
//

import UIKit


class MultiView: UIView {
    @IBOutlet weak var title: UILabel!
    @IBOutlet weak var spinner: UIActivityIndicatorView!
    @IBOutlet weak var car: UIImageView!
    
    static func newFromNib() -> MultiView {
        let views = NSBundle(forClass: MultiView.self).loadNibNamed("MultiView", owner: self, options: nil) as! [MultiView]
        return views.first!
    }
}
